README.md

```diff
@@ -75,6 +75,27 @@ Run the specs:
 $ ./vendor/bin/phpspec run
 ```
 
+### Python
+
+```
+$ cd python
+```
+
+Install, create and activate the virtualenv:
+
+```
+$ pip3 install virtualenv
+$ virtualenv -p python3 venv
+$ . venv/bin/activate
+$ pip install -r requirements.txt
```

Comment:
should probably be `pip3` as with the first `pip` ?

Reply:
it installs the libraries inside the virtual env, no need for `pip3` here

Reply:
the `activate` should be `source ./venv/bin/activate` it seems